imap_processing/glows/ancillary/l1b_conversion_table_v001.json

```
@@ -0,0 +1,42 @@
+{
+    "description": "Table for conversion/decoding ancillary parameters collected onboard by IMAP/GLOWS",
+    "version": "0.1",
```

Do you want to track the version in the filename?
```suggestion
    "version": "001",
```

---

Yes, good call. I renamed the file but it should match the other file versioning format, I think